introduction.md

```diff
@@ -1,3 +1,3 @@
-hi! more text since code school wants a sentence. let's add some copy pasta 
-hi! more text since code school wants a sentence.
-my favorite code school path is JavaScript!
\ No newline at end of file
+*Git
+*JavaScript
+*Ruby
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?